expr/parse_filterql_test.go

```diff
@@ -0,0 +1,108 @@
+package expr
```

Comment:
Maybe add some tests against bad filterql as well to make sure it fails properly?